fixtures/features/api-elements/warnings.json

```diff
@@ -186,42 +186,6 @@
             "attributes": {},
             "content": [
               [
-                99,
-                23
-              ]
-            ]
-          }
-        ]
-      },
-      "content": "Authentication information is not yet supported"
-    },
-    {
-      "element": "annotation",
-      "meta": {
-        "classes": [
-          "warning"
-        ],
-        "links": [
-          {
-            "element": "link",
-            "meta": {},
-            "attributes": {
-              "relation": "origin",
-              "href": "http://docs.apiary.io/validations/swagger#refract-not-supported"
-            },
-            "content": []
-          }
-        ]
-      },
-      "attributes": {
-        "code": 3,
-        "sourceMap": [
-          {
-            "element": "sourceMap",
-            "meta": {},
-            "attributes": {},
-            "content": [
-              [
```

Comment:
This is because authentication is supported now and thus the warnings vanish

On Thu, May 5, 2016, 17:09 Stephen Mizell <notifications@github.com> wrote:

> In fixtures/features/api-elements/warnings.json
> <https://github.com/apiaryio/swagger-zoo/pull/16#discussion_r62200686>:
>
> > -              "relation": "origin",
> > -              "href": "http://docs.apiary.io/validations/swagger#refract-not-supported"
> > -            },
> > -            "content": []
> > -          }
> > -        ]
> > -      },
> > -      "attributes": {
> > -        "code": 3,
> > -        "sourceMap": [
> > -          {
> > -            "element": "sourceMap",
> > -            "meta": {},
> > -            "attributes": {},
> > -            "content": [
> > -              [
>
> This seems to just be removing a lot of things. Any reason for this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apiaryio/swagger-zoo/pull/16/files/e246e1d794724fb1f6c8c47abc9b1aff755d665b#r62200686>
>